### PR TITLE
Add mastodon profiles to user card display

### PR DESCRIPTION
**Pull request by nickynicolson**
As per issue #521

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -7219,19 +7219,20 @@ nickynicolson:
...
   website: https://www.kew.org/science/our-science/people/nicky-nicolson
   city: London
   country: United Kingdom
   expertise:
   - Biodiversity informatics
   - Automation
   - Continuous integration
-  - Document processing
+  - Document proc
+  longitude: -0.1440550845essing
```
Comment (bebatut) [inline _data/people.yaml:7231]:
```suggestion
  - Document processing
```

File: _data/people.yaml [hunk 2]
```diff
@@ -7219,19 +7219,20 @@ nickynicolson:
...
   bio: Software developer turned researcher, learning how to apply software development
     practices to research management
   country_3: GBR
-  continent: Europe
-  longitude: -0.14405508452768728
+  continent: Europe2768728
```
Comment (bebatut) [inline _data/people.yaml:7235]:
```suggestion
  continent: Europe
  longitude: -0.14405508452768728
```

File: _includes/people.html [hunk 3]
```diff
@@ -21,6 +21,10 @@
           {% if user.twitter %}
           <a class="people-card-meta-twitter" href="https://twitter.com/{{ user.twitter }}"><i class="fab fa-twitter"></i> @{{ user.twitter }}</a><br>
           {% endif %}
+          {% if user.mastodon %}
+          <a class="people-card-meta-mastodon" href="https://mastodon.social/@{{ user.mastodon }}"><i class="fab fa-mastodon"></i>
```
Comment (bebatut) [inline _includes/people.html:25]: You probably also need to give the mastodon server. Not everyone is on mastodon.social. I am on piaille.fr for example: https://piaille.fr/@bebatut

Comment (NPDebs) [reply _includes/people.html:25]: Ah, thanks a lot! I saw some misleading info about `mastadon.social` being the default. I'll revert now!

**Review by NPDebs — APPROVED** [review]
Thanks a bunch for the work, @nickynicolson! 🎉 

If you're wondering why there's so many additions, that's because:
1. The `upstream` has changed massively since opening this PR.
2. I tweaked a few things (e.g removing the server address from the card to prevent overflow, etc).

It looks good to be merged. 💯 
@bebatut - A second opinion from you would be appreciated.

**Review by bebatut — APPROVED** [review]

**Pull request merged**